l2tscaffolder/scaffolders/interface.py

```diff
@@ -116,6 +116,17 @@ def __init__(self):
     super(Scaffolder, self).__init__()
     self._output_name = ''
 
+  @abc.abstractmethod
+  def AddEntriesToInitFiles(self) -> Iterator[Tuple[str, str]]:
+    """Returns a list of init files that were modified.
```

Comment:
Please update the docstring to describe what this method is supposed to do, not what it's supposed to return